app/safe/internal/server/route/receive/receive.go

```diff
@@ -85,6 +87,12 @@ func Keys(cid string, w http.ResponseWriter, r *http.Request, spiffeid string) {
 	keysCombined := agePrivateKey + "\n" + agePublicKey + "\n" + aesCipherKey
 	state.SetRootKey(keysCombined)
 
+	if env.RootKeyInputModeManual() && env.ManualRootKeyUpdatesKubernetesSecret() {
+		if err := bootstrap.PersistKeys(agePrivateKey, agePublicKey, aesCipherKey); err != nil {
+			log.ErrorLn(&cid, "Keys: Problem persisting keys", err.Error())
+		}
+	}
```

Comment:
We don’t need the checks in the first `if`.

The user can change the key (at their own risk) if they want to.

"manual mode" is only for the bootstrapping flow.

Can we replace it with this, please?

```go
	if env.ManualRootKeyUpdatesKubernetesSecret() {
 		if err := bootstrap.PersistKeys(agePrivateKey, agePublicKey, aesCipherKey); err != nil {
 			log.ErrorLn(&cid, "Keys: Problem persisting keys", err.Error())
 		}
 	}
```